src/app/test_executive/zkapps.ml

```diff
@@ -36,7 +36,6 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
           work_delay = Some 1
         ; transaction_capacity =
             Some Runtime_config.Proof_keys.Transaction_capacity.small
-        ; block_window_duration_ms = Some 150000
```

Comment:
btw were the changes in this file supposed to happen? They seem unrelated

Reply:
same question with a number of other files here : o

Reply:
Thanks for taking a look!

CI was giving an error that this branch couldn't merge cleanly into develop because it didn't know how to update zkapps.ml file (no idea why, because we did not make any changes on that file). Thus I brought the current version of this file in the develop branch so that it will be the same when the bump-for-lookup-features branch merges. But I can revert that change in the parent branch, or keep an eye open that the changes do not get affected when the final merge is produced.